.ContinuousIntegrationScripts/installVm.sh

```diff
@@ -4,14 +4,16 @@ set -euo pipefail
 VM_VERSION="201911012148"
 BASE_VM_DOWNLOAD_PATH="https://github.com/OpenSmalltalk/opensmalltalk-vm/releases/download/$VM_VERSION"
 
-echo "Installing VM $VM_VERSION for $TRAVIS_OS_NAME"
+echo "Installing VM $VM_VERSION for $RUNNER_OS"
 
 installVmLinux() {
-  VM_FILENAME="squeak.cog.spur_linux64x64_$VM_VERSION"
+  sudo apt-get update
+  sudo apt-get install pulseaudio
+  VM_FILENAME="squeak.cog.spur_linux64x64_itimer_$VM_VERSION"
```

Comment:
github actions does not allow to change the ulimit configuration, so we need to use the itimer vm, I am not sure about the difference.